build.gradle

```diff
@@ -129,6 +129,15 @@ def reobfArtifact = artifacts.add('default', reobfFile) {
     type 'jar'
     builtBy 'reobfJar'
 }
+
+
+task deobfJar(type: Jar) {
```

Comment:
Feel free to ignore this part. it just makes a jar that you can also use while making other mods. I used this to mess with forge to find the lighting bugs